include/alpaka/platform/PlatformUniformCudaHipRt.hpp

```diff
@@ -134,11 +134,11 @@ namespace alpaka
             {
                 ALPAKA_DEBUG_FULL_LOG_SCOPE;
 
-                std::size_t constexpr kiB(1024);
-                std::size_t constexpr miB(kiB * kiB);
+                constexpr std::size_t KiB{1024};
```

Comment:
Is there a reason for preferring `KiB{1024}` over `KiB = std::size_t{1024}`?

Reply:
I did that out of force of habit, sorry!
Both variables should be `auto` now.